components-api/src/main/scala/pl/touk/nussknacker/engine/api/json/TypingType.scala

```diff
@@ -0,0 +1,29 @@
+package pl.touk.nussknacker.engine.api.json
+
+import io.circe._
+import pl.touk.nussknacker.engine.api.typed.typing
+import pl.touk.nussknacker.engine.api.typed.typing._
+
+private[json] object TypingType extends Enumeration {
```

Comment:
Without changes in code, just extracted from another file that contained 3 different things.